flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/NettyShuffleEnvironmentConfiguration.java

```diff
@@ -122,6 +126,10 @@ public boolean isNetworkDetailedMetrics() {
 		return isNetworkDetailedMetrics;
 	}
 
+	public boolean isReleaseExternallyManagedPartitionsOnConsumption() {
```

Comment:
May be better to unify the method with variable name. 
`isReleasePartititionsOnConsumption| isReleaseBlockingPartitionsOnConsumption| isReleaseExternallyManagedPartitionsOnConsumption`

IMO `isReleasePartititionsOnConsumption` might be simple and proper, no need to concern on whether it is blocking or external.

Reply:
`isReleasePartititionsOnConsumption` is ambiguous though; as it stands we always (now, and in the future) want the environment to automatically release consumed partitions if they are not externally managed.

Reply:
Agree, so might define the var as `isReleaseExternallyManagedPartitionsOnConsumption` instead of current `isReleaseBlockingPartitionsOnConsumption` because it might bring confusion whether `Blocking` is strictly equivalent with `ExternallyManaged` as now and future.

Reply:
yes, thats a good point.